src/client/components/ModalWithFeedback/ModalWithFeedback.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button } from '../Button/Button';
+import './ModalWithFeedback.css';
+import { Card } from '../Card/Card';
+import { EventCardText } from '../EventCardText/EventCardText';
+import Modal from '../Modal/Modal';
+
+function ModalWithFeedback() {
+  return (
+    <div className="feedback-modal">
+      <Modal title="Feedback Form">
+        <div className="event-small-container">
+          <span className="dot" />
+          <div className="eventcardtext">
+            <EventCardText
+              title="Elastic - Week 46"
+              datetime="Wednesday | 11 Nov 20|10:00 CEST"
+              mentor="'Mentor Name'"
+              link="https://slack.com/meeting...."
+            />
+          </div>
+        </div>
+        <div className="feedback-textfield">
+          <Card text="You just attended the study group with student1, student2, student3, student4......." />
```

Comment:
This should be an input where students write their feedback. The text is just placeholder in that input.
Use `useState` locally for that input.